### ISSUE TITLE
Export

### ISSUE THREAD

**Issue by Vareska** [LendoK/Blender_GTA_V_model_importer#25]
Is there a way to export the edited object back out of blender to an odd file so it can be used in gta?

**Comment by BrynnaDaRosa** [comment]
No. If you simply read the other issues you would know that there is no open formats export at this time. 

You can use Sollumz but it doesn't take OpenIV openformats, you would have to get Codewalker and export the needed file to xml formats.
